api/predict.py

```diff
@@ -55,22 +54,9 @@ def post(self):
         args = input_parser.parse_args()
         audio_data = args['audio'].read()
 
-        # clean up from earlier runs
-        if os.path.exists("audio.wav"):
-            os.remove("audio.wav")
-
-        if '.wav' in str(args['audio']):
-            file = open("audio.wav", "wb")
-            file.write(audio_data)
-            file.close()
-        else:
-            e = BadRequest()
-            e.data = {'status': 'error', 'message': 'Invalid file type/extension'}
```

Comment:
How do we handle exception for invalid file types?

Reply:
I can add the exception back